src/containers/flows/MintDropNFT/helpers.ts

```diff
@@ -37,80 +27,69 @@ export const getDropData = (
 				dropStage,
 				wheelsTotal: saleData.amountForSale,
 				wheelsMinted: saleData.amountSold,
-				maxPurchasesPerUser,
 			} as DropData);
 		} catch (error) {
+			console.log(error);
 			reject(error);
 		}
 	});
 };
 
 const getDropStage = async (
 	zSaleInstance: Instance,
-	library: Web3Provider,
 ): Promise<Stage | undefined> => {
-	const status = await zSaleInstance.getSaleStatus(library.getSigner());
-
+	const status = await zSaleInstance.getSaleStatus();
+	console.log(status);
```

Comment:
Was this console.log intentionally left in for logging ?

Reply:
Will remove it